.eslintrc.json

```diff
@@ -1,20 +1,36 @@
 {
-  "parserOptions": {
-    "ecmaVersion": 2017
-  },
   "extends": [
-    "eslint:recommended"
+    "eslint:recommended",
+    "plugin:security/recommended"
   ],
   "env": {
     "browser": true,
     "commonjs": true,
     "es6": true
   },
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "plugins": [
+    "jsdoc",
+    "json",
+    "security"
+  ],
   "root": true,
   "rules": {
+    "jsdoc/check-types": "error",
```

Comment:
Sadly JSDoc plugin doesn't have a "recommended" rule set, so we have to cherry pick the rules we may care about. But I may try submitting an upstream PR for some semi-logical default rules and see how that goes.